python/ray/serve/_private/deployment_state.py

```diff
@@ -2000,7 +2005,7 @@ def _check_startup_replicas(
                 replica_startup_message = (
                     f"{replica.replica_id} started successfully "
                     f"on node '{replica.actor_node_id}' after "
-                    f"{e2e_replica_start_latency:.1f}s."
+                    f"{e2e_replica_start_latency:.1f}s (PID: {replica.actor_pid})."
                 )
                 if replica.initialization_latency_s is not None:
                     # This condition should always be True. The initialization
```

Comment:
Should we add it to the message below as well?

Reply:
I think it is all one log message?

Reply:
oo I misread it XD